src/components/Navigation/Sections/HeaderSection.js

```diff
@@ -50,18 +50,23 @@ export default class HeaderSection extends Component {
         <Col md={4} className="text-right">
           {!isEmpty(changes) && (
             <ButtonWithTooltip tip="Revert all changes of the current definition">
-              <Button bsStyle="danger" onClick={() => handleRevert()}>
+              <Button bsStyle="danger" className="revertButton" onClick={() => handleRevert()}>
```

Comment:
CSS classes seem to shy away from camelCase and go towards `-` (i.e., revert-button) and we do that elsewhere. 

Like above, I think we should distinguish between "real" and "identification" classes.